config/protocol.testnet.yml

```diff
@@ -1,6 +1,8 @@
 ProtocolConfiguration:
   Magic: 877933390
+  MaxBlockSystemFee: 150000000000
```

Comment:
No `MaxBlockSize` change? It differs from the default.

Reply:
Yeah, fixed.